jenkins/run

```diff
@@ -154,6 +155,7 @@ ${PODMAN} run \
     --env USER=${USER_NAME} \
     --env WORKSPACE=${WORKSPACE} \
     --env WORKSPACE_TMP=${WORKSPACE_TMP} \
+    --env OIDC_CA="/etc/pki/tls/certs/pbench_CA.crt" \
```

Comment:
I don't think we want to use an environment variable to locate the CA file:  this is almost certainly going to create complications trying to use a common mechanism for deploying the Server for functional testing and Staging/Production.